examples/twittermap/web/public/data/stopwords.js

```diff
@@ -0,0 +1,435 @@
+function external_stop_words() {
```

Comment:
Where did you get these stopwords? I think we need to cite the origin here. For example,

// Copied from http://xxx.xxx ...

Reply:
ok,sure